client/src/components/Error/Error.mui.ts

```diff
@@ -0,0 +1,11 @@
+import { makeStyles } from '@material-ui/core/styles';
+
+export default makeStyles({
+	errorBanner: {
+		backgroundColor: '#e81410',
```

Comment:
This color should be a variable available throughout the client -> to be fixed in a later issue